episode-18-code/commands/addrole.js

```diff
@@ -3,30 +3,28 @@ const errors = require("../utils/errors.js");
 
 module.exports.run = async (bot, message, args) => {
 
-  //!addrole @andrew Dog Person
+  //!addrole @user Dog Person
+
   if (!message.member.hasPermission("MANAGE_ROLES")) return errors.noPerms(message, "MANAGE_ROLES");
-  if (args[0] == "help") {
-    message.reply("Usage: !addrole <user> <role>");
-    return;
-  }
-  let rMember = message.guild.member(message.mentions.users.first()) || message.guild.members.get(args[0]);
+  if (args[0] === "help") return message.reply("Usage: `!addrole @user role");
+  const rMember = message.guild.member(message.mentions.users.first()) || message.guild.members.get(args[0]);
   if (!rMember) return errors.cantfindUser(message.channel);
-  let role = args.join(" ").slice(22);
+  const role = args.join(" ").slice(22);

```

Comment:
Don't use .slice(22):

Reply:
Figured i'd forget some issues already in the code.